ui/apps/everest/src/pages/settings/monitoring-endpoints/createEditModal/create-edit-modal.tsx

```diff
@@ -1,18 +1,18 @@
-import { useMemo } from 'react';
```

Comment:
why we need to cut this part of code and add the same below? please try pnpm format in repo before commit

Reply:
Sorry, i wont do this again(